src/scss/constants.scss

```diff
@@ -0,0 +1,20 @@
+$base-padding: 10%;
+
+
+$text-size: 16px;
+$header-size: 24px;
+
+$base-color: rgb(17, 12, 92);
+$second-color: rgb(92, 1, 104);
+$nav-link-color: rgb(214, 214, 214);
+$nav-link-active-color: #fff;
```

Comment:
It's better to store your colors in one form. And, as I know, a good way is to write colors in the `hex` format (`#fff`).